src/Octoshift/AdoClient.cs

```diff
@@ -133,6 +148,44 @@ public virtual async Task<JArray> GetWithPagingAsync(string url, string continua
             return data;
         }
 
+        public virtual async Task<IEnumerable<T>> GetWithPagingTopSkipAsync<T>(string url, Func<JToken, T> selector) => await GetWithPagingTopSkipAsync(url, 0, selector);
+
+        public virtual async Task<IEnumerable<T>> GetWithPagingTopSkipAsync<T>(string url, int skip, Func<JToken, T> selector)
+        {
+            if (string.IsNullOrWhiteSpace(url))
```

Comment:
nit: We do have an extension method for this
```suggestion
            if (url.IsNullOrWhiteSpace())
```